.github/workflows/actions-build.yml

```diff
@@ -1,34 +1,51 @@
-name: 'Terraform Build Actions'
+name: "Terraform Build Actions"
 
 on:
   pull_request:
     branches:
-      - 'develop'
+      - "develop"
   push:
     branches:
-      - 'develop'
-  
+      - "develop"
+
 jobs:
+  print-version:
+    runs-on: ubuntu-latest
+    name: A job to print the version
+    outputs:
+      version: ${{ steps.get-version.outputs.version }}
+    steps:
+      - name: checkout repository
+        uses: actions/checkout@v3
+      - name: test action
+        id: get-version
+        uses: beaconbrigade/package-json-version@v0.3
+        with:
+          path: .
+      - name: Execute
+        run: echo ${{ steps.get-version.outputs.version }} >> "$GITHUB_OUTPUT"
   build:
     runs-on: ubuntu-latest
+    needs: print-version
```

Comment:
The only issue i see is that this isnt passed between the jobs unless we push. to artifactory. Can we call this step in the deploy step and export there?